Source/Core/DolphinQt/Debugger/CodeViewWidget.cpp

```diff
@@ -202,6 +284,7 @@ void CodeViewWidget::Update()
   }
 
   resizeColumnsToContents();
+  setColumnWidth(5, static_cast<int>(WIDTH_PER_BRANCH_ARROW * m_branches.size()));
```

Comment:
Thoughts about naming the columns via e.g. an enum to make the code more obvious?

Reply:
Might as well. Do we prefer an enum or several constexpr ints for values that are going to be used as integers?

Reply:
For this I usually go for an unnamed enum in a namespace, but I don't know what the project prefers.

Reply:
We prefer enum classes over plain enums, which I suppose would exclude unnamed enums.